docs/ab-testing.md

```diff
@@ -139,6 +139,14 @@ define([
     })
 ```
 
+### Forcing yourself into a test
+Add #ab-<TestName>=<VariantName> to the end of your URL (in dev or prod) to force yourself into a test.
+e.g. www.theguardian.com/news#ab-MyGreatTest=GreenButton
+
+#### Firing complete events in dev mode
+In prod, the completion events are fired based on the MVT ID cookie. This doesn't exist in dev, so if you need to test a complete event, follow the #ab- pattern above.
```

Comment:
> This doesn't exist in dev, so if you need to test a complete event, follow the #ab- pattern above.

Could this be slightly clearer ? Maybe
`This doesn't exist in dev, so if you need to test a complete event, force yourself into the test using the #ab-<TestName>=<VariantName> pattern described above.`